internal/cmd/root.go

```diff
@@ -188,7 +189,10 @@ func display(notifications notifications.Notifications) error {
 }
 
 func displayTable(n notifications.Notifications) {
```

Comment:
Following

https://github.com/nobe4/gh-not/pull/234#discussion_r1902895374

Here, using fmt.Fprintf(os.Stout improve the readability by avoiding to use too much nolint